xml/storage_multipath.xml

```diff
@@ -1131,393 +1183,300 @@
     and whether the LUNs are correctly assigned to the server.
    </para>
    <para>
-    If the LUNs are seen by the HBA driver, but there are no corresponding
-    block devices, additional kernel parameters are needed to change the SCSI
-    device scanning behavior, such as to indicate that LUNs are not numbered
-    consecutively. For information, see <citetitle>TID 3955167: Troubleshooting
+    If LUNs are seen by the HBA driver, but no corresponding block devices
+    are created, additional kernel parameters may be needed. See <citetitle>TID 3955167: Troubleshooting
     SCSI (LUN) Scanning Issues</citetitle> in the &suse; Knowledgebase at
     <link xlink:href="https://www.suse.com/support/kb/doc.php?id=3955167"/>.
    </para>
   </sect2>
 
   <sect2 xml:id="sec-multipath-configuration-partitioning">
-   <title>Partitioning multipath devices</title>
-   <para>
-    Partitioning devices that have multiple paths is not recommended, but it is
-    supported. You can use the <command>kpartx</command> tool to create
-    partitions on multipath devices without rebooting. You can also partition
-    the device before you attempt to configure multipathing by using the
-    Partitioner function in &yast;, or by using a third-party partitioning
-    tool.
-   </para>
-   <para>
-    Multipath devices are device-mapper devices. Modifying device-mapper
-    devices with command line tools (such as parted, kpartx, or fdisk) works,
-    but it does not necessarily generate the udev events that are required to
-    update other layers. After you partition the device-mapper device, you
-    should check the multipath map to make sure the device-mapper devices were
-    mapped. If they are missing, you can remap the multipath devices or reboot
-    the server to pick up all of the new partitions in the multipath map.
-   </para>
-   <para>
-    The device-mapper device for a partition on a multipath device is not the
-    same as an independent device. When you create an LVM logical volume using
-    the whole device, you must specify a device that contains no partitions. If
-    you specify a multipath partition as the target device for the LVM logical
-    volume, LVM recognizes that the underlying physical device is partitioned
-    and the create fails. If you need to subdivide a SAN device, you can carve
-    LUNs on the SAN device and present each LUN as a separate multipath device
-    to the server.
-   </para>
+    <title>Partitions on multipath devices</title>
+    <para>
+      Multipath maps can have partitions like their path devices.
+      Partition table scanning and device node creation for partitions
+      is done in user space by the
+      <command>kpartx</command> tool. <command>kpartx</command> is
+      automatically invoked by udev rules; there is usually no need to run it
+      manually. Technically, &ldquo;partition&rdquo; devices created by kpartx are
```

Comment:
```suggestion
      manually. Technically, &ldquo;partition&rdquo; devices created by <command>kpartx</command> are
```

Reply:
Is there any term that can be used without the quotes?

Reply:
None that would describe these devices better.